src/cunumeric/index/wrap.cc

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2022 NVIDIA Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+#include "cunumeric/index/wrap.h"
+#include "cunumeric/index/wrap_template.inl"
+
+namespace cunumeric {
+
+using namespace Legion;
+using namespace legate;
+
+template <int DIM>
+struct WrapImplBody<VariantKind::CPU, DIM> {
+  void operator()(const AccessorWO<Point<DIM>, 1>& out,
+                  const Pitches<0>& pitches_out,
+                  const Rect<1>& out_rect,
+                  const Pitches<DIM - 1>& pitches_in,
+                  const Rect<DIM>& in_rect,
+                  const bool dense) const
+  {
+    const size_t start     = out_rect.lo[0];
```

Comment:
I think Point coordinates are `int64_t`, not `size_t`. Is there a need to use `size_t`?